lib/ruby_lsp/requests/workspace_symbol.rb

```diff
@@ -0,0 +1,76 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Requests
+    # ![Workspace symbol demo](../../workspace_symbol.gif)
+    #
+    # The [workspace symbol](https://microsoft.github.io/language-server-protocol/specification#workspace_symbol)
+    # request allows fuzzy searching declarations in the entire project. On VS Code, use CMD + T to search for symbols.
```

Comment:
Since `CMD` is Mac-specific, I think it we should either also include the Windows/Linux shortcuts, or not state it at all.